src/ImageSharp/Formats/Jpg/Components/DCT.cs

```diff
@@ -0,0 +1,340 @@
+            Vector4 mb0 = (my1 * C_1_501321) + mz0 + mz3;
+
+            Vector4 my2 = s.V2R;
+            Vector4 my6 = s.V6R;
+            mz4 = (my2 + my6) * C_0_541196;
+            Vector4 my0 = s.V0R;
+            Vector4 my4 = s.V4R;
+            mz0 = my0 + my4;
+            mz1 = my0 - my4;
+
+            mz2 = mz4 + (my6 * C_1_847759);
+            mz3 = mz4 + (my2 * C_0_765367);
+
+            my0 = mz0 + mz3;
+            my3 = mz0 - mz3;
+            my1 = mz1 + mz2;
+            my2 = mz1 - mz2;
+
+            d.V0R = my0 + mb0;
+            d.V7R = my0 - mb0;
+            d.V1R = my1 + mb1;
+            d.V6R = my1 - mb1;
+            d.V2R = my2 + mb2;
+            d.V5R = my2 - mb2;
+            d.V3R = my3 + mb3;
+            d.V4R = my3 - mb3;
+        }
+
+        /// <summary>
+        /// Original:
+        /// <see>
+        ///     <cref>https://github.com/norishigefukushima/dct_simd/blob/master/dct/dct8x8_simd.cpp#L15</cref>
+        /// </see>
+        /// </summary>
+        /// <param name="s">Source</param>
+        /// <param name="d">Destination</param>
+        public static void FDCT8x4_LeftPart(ref Block8x8F s, ref Block8x8F d)
+        {
+            Vector4 c0 = s.V0L;
+            Vector4 c1 = s.V7L;
+            Vector4 t0 = c0 + c1;
+            Vector4 t7 = c0 - c1;
+
+            c1 = s.V6L;
+            c0 = s.V1L;
+            Vector4 t1 = c0 + c1;
+            Vector4 t6 = c0 - c1;
+
+            c1 = s.V5L;
+            c0 = s.V2L;
+            Vector4 t2 = c0 + c1;
+            Vector4 t5 = c0 - c1;
+
+            c0 = s.V3L;
+            c1 = s.V4L;
+            Vector4 t3 = c0 + c1;
+            Vector4 t4 = c0 - c1;
+
+            c0 = t0 + t3;
+            Vector4 c3 = t0 - t3;
+            c1 = t1 + t2;
+            Vector4 c2 = t1 - t2;
+
+            d.V0L = c0 + c1;
+            d.V4L = c0 - c1;
+
+            Vector4 w0 = new Vector4(0.541196f);
+            Vector4 w1 = new Vector4(1.306563f);
+
+            d.V2L = (w0 * c2) + (w1 * c3);
+            d.V6L = (w0 * c3) - (w1 * c2);
+
+            w0 = new Vector4(1.175876f);
+            w1 = new Vector4(0.785695f);
+            c3 = (w0 * t4) + (w1 * t7);
+            c0 = (w0 * t7) - (w1 * t4);
+
+            w0 = new Vector4(1.387040f);
+            w1 = new Vector4(0.275899f);
+            c2 = (w0 * t5) + (w1 * t6);
+            c1 = (w0 * t6) - (w1 * t5);
+
+            d.V3L = c0 - c2;
+            d.V5L = c3 - c1;
+
+            Vector4 invsqrt2 = new Vector4(0.707107f);
+            c0 = (c0 + c2) * invsqrt2;
+            c3 = (c3 + c1) * invsqrt2;
+
+            d.V1L = c0 + c3;
+            d.V7L = c0 - c3;
+        }
+
+        /// <summary>
+        /// Original:
+        /// <see>
+        ///     <cref>https://github.com/norishigefukushima/dct_simd/blob/master/dct/dct8x8_simd.cpp#L15</cref>
+        /// </see>
+        /// </summary>
+        /// <param name="s">Source</param>
```

Comment:
Actually I haven't hit that rule at all. It should trigger on parameter docs, class docs. Maybe they've changed the spec for it. Ahhh, it's been disabled, see [here](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/KnownChanges.md).